roles/apache2/vars/main.yaml

```
@@ -10,22 +10,8 @@ _a2_conffile_owner: |-
 _a2_conffile_group: |-
   {{ apache2__conffile_group or 'root' }}
 
-_a2_listen_http: |-
-  {{
-    []
-    if not apache2__listen_http else
-    [apache2__listen_http]
-    if apache2__listen_http is not iterable or apache2__listen_http is string
-    else apache2__listen_http
-  }}
-_a2_listen_https: |-
-  {{
-    []
-    if not apache2__listen_https else
-    [apache2__listen_https]
-    if apache2__listen_https is not iterable or apache2__listen_https is string
-    else apache2__listen_https
-  }}
+_a2_listen_http: '{{ apache2__listen_http | d([]) | tina_pm.common.to_list }}'
```

This is not taking into account the case where the var is either null or an empty string:

```
In : to_list(None)
Out: [None]

In : to_list('')
Out: ['']
```

 You need either of these:

```suggestion
_a2_listen_http: '{{ apache2__listen_http | d([], True) | tina_pm.common.to_list }}'
```
```suggestion
_a2_listen_http: '{{ (apache2__listen_http or []) | tina_pm.common.to_list }}'
```
(This second option fails if undefined, but that is not a concern as the var is defined in defaults)